Monika After Story/game/zz_calendar.rpy

```diff
@@ -222,8 +223,12 @@ M̼̤̱͇̤ ͈̰̬͈̭ͅw̩̜͇͈ͅa̲̩̭̩ͅs̙ ̣͔͓͚̰h̠̯̫̼͉e̗̗̮r
             """
             super(renpy.Displayable, self).__init__()
 
+            # dont really feel like changing every image line to not do inline
+            # if statements so this will work for now.
+            self.day_mode = not mas_globals.dark_mode
```

Comment:
Calendar was not affected by `mas_globals.dark_mode` as it is not considered as part of the ui but part of the background, so it should change depending on the day or the filter cycle.

Reply:
but why? calendar is all UI elements so it should follow the UI rules.

Reply:
Calendar is only somewhat a button. Since it hangs on the spaceroom walls and also functions as a part of the room, we treat it as such. Additionally, it makes no sense for the calendar to change with ui modes because it's a part of the room.